File: java/3264.java
class Solution {
    public int[] getFinalState(int[] nums, int k, int m) {
        int n = nums.length;
        while (--k >= 0) {
            int i = 0;
            for (int j = 1; j < n; j++)
                if (nums[i] > nums[j])
                    i = j;
            nums[i] *= m;
        }
        return nums;
    }
}

File: python3/3264.py
class Solution:
    def getFinalState(self, nums: List[int], k: int, m: int) -> List[int]:
        n = len(nums)
        while k > 0:
            i = 0
            j = n - 1
            while j >= 0:
                if nums[i] >= nums[j]:
                    i = j
                j -= 1
            nums[i] *= m
            k -= 1
        return nums
